activerecord/lib/active_record/tasks/sqlite_database_tasks.rb

```diff
@@ -64,6 +66,18 @@ def configuration
         def root
           @root
         end
+
+        def run_cmd(cmd, args, out)
+          #fail run_cmd_error(cmd, args) unless Kernel.system(cmd, *args, out: out)
```

Comment:
My guess would be the quoting on `statement`, which isn't required for the splat form.